### PR TITLE
Footer Form

### PR DESCRIPTION
**Pull request by Mhamad-Raad**
in this section, I have added the following functionalities:

- add form to the program, in which the user can contact me through.
- add footer background, which contains the form.
- connect the form to fromspree, which automatically sends me an email after clicking submit.
- add local form validation.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -334,6 +335,21 @@ <h2 class="languages-title ">Skills</h2>
...
       </section>
 
+      <section class="contact-me-bg">
+        <div class="contact-me">
+          <h2 class="contact-me-title-footer">Contact Me</h2>
+            <p class="contact-me-description">
+              If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+            </p>
+            <form class="contact-me-form" action="https://formspree.io/f/xoqbvvly" method="POST">
+              <input type="name" name="name" placeholder="full name" class="contact-me-input" maxlength="30" required>
```
Comment (ichala) [inline index.html]: - [x] kindly update your input type which should be `text` and not `name` , after that your project will be ready , great job 👏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Mhamad-Raad , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design ✅ 
- Correct workflow ✅ 

 ## Required Changes ♻️

- [ ] **_Check the comments below_**

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mhamad-Raad) [reply index.html]: I have just updated it, can you please check again.

**Review by ghost — APPROVED** [review]
Hi @Mhamad-Raad , 

Good work implementing the change requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**